commons/src/main/java/com/afollestad/materialdialogs/folderselector/FileChooserDialog.java

```diff
@@ -192,6 +193,16 @@ public void onSelection(MaterialDialog materialDialog, View view, int i, CharSeq
         }
     }
 
+
+    private void checkIfCanGoUp() {
+        try {
+            String path = parentFolder.getPath().split("/")[1];
```

Comment:
this code will try to split the path and get the first array index. for example, /data/media will split as [0] null | [1] data | [2] media
if the directory is "/", we cannot access to [1] index and it will return an error handled from the catch block that will set the canGoUp boolean as false, because you are in the root path